suites/squid/common/regression/rados/rados_3-AZ-Cluster.yaml

```diff
@@ -0,0 +1,84 @@
+# 3 AZ cluster deployment tests
```

Comment:
@vamahaja Is it ok to add this FG-specific folder structure?

Reply:
@sunilangadi2 create fg specific directory under regression and use for regrssion suites.